sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java

```diff
@@ -126,13 +158,23 @@ public FavoritesService favorites() {
   }
 
   @Override
-  public QualityProfilesService qualityProfiles() {
-    return qualityProfilesService;
+  public QualityProfilesService qualityProfilesOld() {
+    return qualityProfilesOld;
+  }
+
+  @Override
+  public QualityprofilesService qualityProfiles() {
+    return qualityprofiles;
+  }
+
+  @Override
+  public org.sonarqube.ws.client.issue.IssuesService issuesOld() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "IssuesService"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)